src/etc/inc/services.inc

```diff
@@ -841,7 +844,12 @@ EOPP;
 				$dhcpdconf .= "		$deny_action dynamic bootp clients;\n";
 			}
 
-			if (isset($poolconf['denyunknown'])) {
+			// set pool MAC limitations
+			if ($poolconf['denyunknown'] == "class") {
+				$dhcpdconf .= "		allow members of \"s_{$dhcpif}\";\n";
+				$dhcpdconf .= "		$deny_action known-clients;\n";
+				$dhcpdconf .= "		$deny_action unknown-clients;\n";
+			} else if (isset($poolconf['denyunknown'])) {
```

Comment:
This field has changed from a boolean checkbox to a select form. IT must be checked comparing its value with the string `"disabled"` from now.

Also, old configuration items should be converted to the new field type.